src/scripts/updateStaticData.ts

```diff
@@ -0,0 +1,30 @@
+import { AppDataSource } from "..";
+import { config } from "../config";
+import { makeLogger } from "../logger";
+
+import { Country, Region } from "../entity/Country";
+import { getWorld } from "../wrapper/wrapper";
+
+const logger = makeLogger(module);
+
+export async function updateStaticData() {
+    try {
+        logger.info("Updating static map data...");
```

Comment:
since its static seems kind of silly to update it at all.

Reply:
It needs to be loaded into the db
And if there'll ever be a change to the map it'll need to be updated